typings-test.ts

```diff
@@ -1,554 +0,0 @@
-// tslint:disable-next-line:no-implicit-dependencies
```

Comment:
I noticed running `npm run lint` produces 

```
> immutability-helper@2.8.1 lint /Users/mkolodny/projects/immutability-helper
> tslint -p tsconfig.test.json

No valid rules have been specified for TypeScript files
```

Is there something we should be doing instead?

Reply:
Looks like [a TSLint issue](https://github.com/palantir/tslint/issues/3735), but it's resolved in 67360db. Thanks for pointing this out!